### PR TITLE
Spexapc 956 egress gtw ha

### PR DESCRIPTION
**Pull request by LubosRachel**

### CODE REVIEW

File: charts/cilium-egress-gtw-ha-ips/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+{{- range .ip }}
+  - nodeSelector:
+      matchLabels:
+        egressgtw: "true"
+    egressIP: {{ . }}
+  {{- end }}
+---
+{{- end }}
+{{- end }}
+```
```
Comment (pstefka) [inline charts/cilium-egress-gtw-ha-ips/README.md]: Please remove - can be checked in the templates folder

File: charts/cilium-egress-gtw-ha-ips/templates/IsovalentEgressGatewayPolicy.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+spec:
+  destinationCIDRs:
+    - 0.0.0.0/0
+  selectors:
+    - podSelector:
+        matchLabels:
+          io.kubernetes.pod.namespace: {{ .namespace }}
+  azAffinity: {{ .azAffinity | default $.Values.azAffinity | default "localOnlyFirst" }}
+egressGroups:
+{{- range .ip }}
```
Comment (pstefka) [inline charts/cilium-egress-gtw-ha-ips/templates/IsovalentEgressGatewayPolicy.yaml]: Please change to plural - ips

File: charts/cilium-egress-gtw-ha-ips/README.md
Comment (pstefka) [inline charts/cilium-egress-gtw-ha-ips/README.md:1]: Please use a linter .. e.g. DavidAnson.vscode-markdownlint in vscode

**Review by pstefka — CHANGES_REQUESTED** [review]
Some

File: charts/cilium-egress-gtw-ha-ips/README.md (hunk 1 above)
Comment (LubosRachel) [reply charts/cilium-egress-gtw-ha-ips/README.md]: removed

File: charts/cilium-egress-gtw-ha-ips/templates/IsovalentEgressGatewayPolicy.yaml (hunk 2 above)
Comment (LubosRachel) [reply charts/cilium-egress-gtw-ha-ips/templates/IsovalentEgressGatewayPolicy.yaml]: changed to plural - ips

File: charts/cilium-egress-gtw-ha-ips/README.md
Comment (LubosRachel) [reply charts/cilium-egress-gtw-ha-ips/README.md:1]: linter used. Thanks for the suggestion on linter extension 👍

File: index.yaml [hunk 3]
```diff
@@ -28,17 +28,46 @@ entries:
...
+    digest: 409f3ee56222bc9b68b0f92e28b90d2613fd00b5b01ff75205584a1df3899f44
+    keywords:
+    - rancher
+    - cilium
+    - enterprise
+    - egress
+    name: cilium-egress-gtw-ha-ips
+    urls:
+    - packaged_charts/cilium-egress-gtw-ha-ips-1.0.0.tgz
+    version: 1.0.0
```
Comment (pstefka) [inline index.yaml]: Please remove v1.0.0 (not used), i.e. remove packaged_chart v1.0.0 and re-run build script

File: packaged_charts/cilium-egress-gtw-ha-ips-1.0.0.tgz
Comment (pstefka) [inline packaged_charts/cilium-egress-gtw-ha-ips-1.0.0.tgz:1]: please remove

**Review by pstefka — APPROVED** [review]
After v1.0.0 removal -> approved. Nice work!

File: index.yaml (hunk 3 above)
Comment (LubosRachel) [reply index.yaml]: old version v1.0.0 removed , build script has been re-run

File: packaged_charts/cilium-egress-gtw-ha-ips-1.0.0.tgz
Comment (LubosRachel) [reply packaged_charts/cilium-egress-gtw-ha-ips-1.0.0.tgz:1]: removed

**Review by pstefka — APPROVED** [review]
LGTM! 👌

**Pull request merged**